### PR TITLE
CTOR 775: Fix apps::backup::rubrik::restapi::mode::jobs long output when a job is not detected

### PR DESCRIPTION
**Pull request by itoussies**
## Description

Refresh Rubrik Rest API following the plugins changes.

Needs https://github.com/centreon/centreon-plugins/pull/5171 to be merged before merging this one.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [X] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ title: Rubrik Rest API
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Dépendances du Connecteur de supervision
+
+Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Rubrik Rest API** 
+depuis l'interface web et le menu **Configuration > Gestionnaire de connecteurs de supervision** :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:11]:
```suggestion
depuis la page **Configuration > Gestionnaire de connecteurs de supervision** :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 2]
```diff
@@ -380,29 +387,36 @@ telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
...
-    --proto='https' \
-    --port='443' \
-    --proxyurl='http://myproxy.mycompany.org:8080' \
-    --api-password='****' \
-    --api-username='centreon' \
-    --verbose
+	--plugin=apps::backup::rubrik::restapi::plugin \
+	--mode=jobs \
+	--hostname='10.0.0.1' \
+	--api-username='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:393]: mandatory?

File: pp/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ title: Rubrik Rest API
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Connector Dependencies
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:8]:
```suggestion
## Connector dependencies
```

File: pp/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 4]
```diff
@@ -373,30 +380,36 @@ Once the plugin is installed, log into your Centreon poller's CLI using the
...
-    --port='443' \
-    --proxyurl='http://myproxy.mycompany.org:8080' \
-    --api-password='****' \
-    --api-username='centreon' \
-    --verbose
+/usr/lib/centreon/plugins/centreon_rubrik_restapi.pl \
+	--plugin=apps::backup::rubrik::restapi::plugin \
+	--mode=jobs \
+	--hostname='10.0.0.1' \
+	--api-username='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:387]: mandatory?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 5]
```diff
@@ -57,14 +63,14 @@ pour en savoir plus sur la découverte automatique de services et sa [planificat
 
 ### Métriques & statuts collectés
 
-Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.
+Voici le tableau des services pour ce connecteur, détaillant les métriques et statuts rattachées à chaque service.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:66]:
```suggestion
Voici le tableau des services pour ce connecteur, détaillant les métriques et statuts rattachés à chaque service.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 6]
```diff
@@ -238,13 +245,16 @@ yum install centreon-plugin-Applications-Rubrik-Restapi
...
+| RUBRIKAPISECRET         | Service account secret (Mandatory with API service account ID for the second authentication way)                                                   |                   | X           |
+| RUBRIKAPITOKEN          | Use token authentication (the third way to authenticate). If option is empty, token is created.                                                    |                   | X           |
+| RUBRIKAPIPROTO          | Specify https if needed                                                                                                         | https             |             |
+| RUBRIKAPIPORT           | Port used                                                                                                                            | 443               |             |
+| RUBRIKAPIEXTRAOPTIONS   | Any extra option you may want to add to every command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:257]:
```suggestion
| Macro                   | Description                                                                                                                                        | Valeur par défaut | Obligatoire |
|:------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| RUBRIKAPIUSERNAME       | API username (Mandatory with API password for the first authentication way)                                                                        |                   | X           |
| RUBRIKAPIPASSWORD       | API password (Mandatory with API username for the first authentication way)                                                                        |                   | X           |
| RUBRIKAPISERVICEACCOUNT | Service account ID (Mandatory with API service account secret for the second authentication way)                                                   |                   | X           |
| RUBRIKAPISECRET         | Service account secret (Mandatory with API service account ID for the second authentication way)                                                   |                   | X           |
| RUBRIKAPITOKEN          | Use token authentication (the third way to authenticate). If option is empty, token is created.                                                    |                   | X           |
| RUBRIKAPIPROTO          | Specify https if needed                                                                                                         | https             |             |
| RUBRIKAPIPORT           | Port used                                                                                                                            | 443               |             |
| RUBRIKAPIEXTRAOPTIONS   | Any extra option you may want to add to every command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). |                   |             |
```

**Comment by lucie-dubrunfaut** [comment]
Rebase here https://github.com/centreon/centreon-documentation/pull/3637

**Pull request closed without merging**